pyproject.toml

```diff
@@ -16,6 +16,10 @@ sqlalchemy = "^1.3"
 pytest = "^5.3"
 docker = { version = "^4.1", optional = true }
 reasonable = { version = "^0.1.6", optional = true }
+pyshacl = "^0.11.4"
+
+[tool.poetry.scripts]
+brick_validate = "brickschema.validate:__main"
```

Comment:
Ah, I see how you're doing the binary script now. What do you think about having the script go into a dedicated `bin/` directory? It can import everything it needs from the `brickschema` package